index.js

```diff
@@ -1,16 +1,37 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
 
+const header = document.querySelector("#header")
+console.log("Here's your header: ", header)
 
 /***** Deliverable 2 *****/
 
+header.style.color = "red"
 
 /***** Deliverable 3 *****/
-// console.log("the player object looks like this:", player)
+console.log("the player object looks like this:", player)
 
+const name = document.querySelector("h2")
+name.textContent = player.name 
 
-/***** Deliverable 4 *****/
+const nickname = document.querySelector("em")
+nickname.textContent = player.nickname
 
+const photo = document.querySelector("img")
+photo.src = player.photo
+photo.alt = player.name
 
-/***** Deliverable 5 *****/
\ No newline at end of file
+/***** Deliverable 4 *****/
+const array = player.goals
+const ul = document.querySelector("ul")
+array.forEach(function(goal){
+    const li = document.createElement("li")
+    li.dataset.id = goal.id
+    li.innerHTML = `<p>${goal.description}</p>
+    <a href=${goal.link} target="_blank">${goal.link}</a>`
+    ul.append(li)
+})
```

Comment:
It might be helpful to wrap this code up into a function, that will help clean up the code and make it easier if you need to display other players as well - but overall the approach looks good!